tracing-api/src/main/java/com/palantir/tracing/api/TraceHttpHeaders.java

```diff
@@ -39,4 +39,14 @@ public interface TraceHttpHeaders {
      */
     @Deprecated
     String ORIGINATING_SPAN_ID = "X-OrigSpanId";
+
+    /**
+     * For traces initiated with span type {@link SpanType#SERVER_INCOMING},
+     * this header will be set to the requestId generated for that trace.
```

Comment:
I'm having trouble parsing this sentence. Perhaps something like:

> The {@code Parent-Request-Id} header represents the requestId of the caller, if present. This provides a direct causal connection between requests up a call stack. This can be an advantage over traceId which provides a broad set of data without information to understand the call pathing unless spans are sampled.